packages/perseus/src/server-item-renderer.tsx

```diff
@@ -541,14 +541,19 @@ const styles = StyleSheet.create({
     },
 });
 
-const ref = React.forwardRef<
+export default React.forwardRef<
     ServerItemRenderer,
     Omit<PropsFor<typeof ServerItemRenderer>, "onRendered">
->((props, ref) => (
-    <LoadingContext.Consumer>
-        {({onRendered}) => (
-            <ServerItemRenderer {...props} onRendered={onRendered} ref={ref} />
-        )}
-    </LoadingContext.Consumer>
-));
-export default ref;
+>(function ServerItemRendererWithRef(props, ref) {
```

Comment:
Some part of my OCD brain wants the function to live outside of this when using the keyword `function`.

Like instead of:

``` TS
export default React.forwardRef<
  ServerItemRenderer,
  PropsFor<typeof ServerItemRenderer>
>(function ServerItemRendererWithRef(props, ref) {
  // ...whatever...
})
```

Do this:

``` TS
function ServerItemRendererWithRef(props, ref) {
  // ...whatever...
}

export default React.forwardRef<
  ServerItemRenderer,
  PropsFor<typeof ServerItemRenderer>
>(ServerItemRendererWithRef)
```

But maybe that's not even possible with `ref` 🤔 

Just a nit/thought, disregard.

Reply:
I'll try, but I think we'll have an easier time with types if we leave it inline.  If it's outside we will likely have to type `ServerItemRendererWithRef` function fully so that it fits in the `forwardRef()`. If we leave it inline, TypeScript helps us because it can infer the `ServerItemRendererWithRef` function's parameter types because it's inside the `forwardRef`.

Reply:
Ok.  We _can_ extract the function, but without additional typing, both `props` and `ref` are `any`. 

I can do this, but this feels uglier than the current syntax:

<img width="582" height="377" alt="image" src="https://github.com/user-attachments/assets/67815607-3092-4512-8c33-673cab1af58d" />

I'm going to land this as-is. Feel free to adjust in a new PR if you have ideas for a better approach.